client/app/scripts/liveblog-edit/posts.service.js

```diff
@@ -93,10 +102,24 @@ define([
                 mainItem: post.groups[1].refs[0],
                 items: post.groups[1].refs
             });
-            // complete Post With User Information
-            userList.getUser(post.original_creator).then(function(user) {
-                post.user = user;
+            post.comments = _.reduce(post.groups[1].refs, function(is, val) {
+                return is || _.isUndefined(val.item.name);
```

Comment:
post gets comments if `val.item.name` is undefined for one of the item? I don't follow the logic :)
What is "name"? Can this expression be more clear ?